### ISSUE TITLE
Unable to login to PI-Hole, got a HTTP status code response '307' instead of '302'

### ISSUE THREAD

**Issue by SiM22** [eko/pihole-exporter#11]
This was working a few weeks back, i am not sure what has changed. I have tried both via IP and DNS, the only redirect if you follow this in a browser/developer tools is a 301, not a 307.

```
pi-hole-exporter    | 2019-08-06T07:51:48.240888700Z 2019/08/06 07:51:48 ------------------------------------
pi-hole-exporter    | 2019-08-06T07:51:48.241230421Z 2019/08/06 07:51:48 -  PI-Hole exporter configuration  -
pi-hole-exporter    | 2019-08-06T07:51:48.241254412Z 2019/08/06 07:51:48 ------------------------------------
pi-hole-exporter    | 2019-08-06T07:51:48.241261895Z 2019/08/06 07:51:48 PIHoleHostname : XXXX
pi-hole-exporter    | 2019-08-06T07:51:48.241268251Z 2019/08/06 07:51:48 PIHolePassword : XXXX
pi-hole-exporter    | 2019-08-06T07:51:48.241435407Z 2019/08/06 07:51:48 Port : 8053
pi-hole-exporter    | 2019-08-06T07:51:48.241697784Z 2019/08/06 07:51:48 Interval : 10s
pi-hole-exporter    | 2019-08-06T07:51:48.241800713Z 2019/08/06 07:51:48 ------------------------------------
pi-hole-exporter    | 2019-08-06T07:51:48.241934490Z 2019/08/06 07:51:48 New Prometheus metric registered: domains_blocked
pi-hole-exporter    | 2019-08-06T07:51:48.242049222Z 2019/08/06 07:51:48 New Prometheus metric registered: dns_queries_today
pi-hole-exporter    | 2019-08-06T07:51:48.242157463Z 2019/08/06 07:51:48 New Prometheus metric registered: ads_blocked_today
pi-hole-exporter    | 2019-08-06T07:51:48.242262395Z 2019/08/06 07:51:48 New Prometheus metric registered: ads_percentag_today
pi-hole-exporter    | 2019-08-06T07:51:48.242359561Z 2019/08/06 07:51:48 New Prometheus metric registered: unique_domains
pi-hole-exporter    | 2019-08-06T07:51:48.242668975Z 2019/08/06 07:51:48 New Prometheus metric registered: queries_forwarded
pi-hole-exporter    | 2019-08-06T07:51:48.242832429Z 2019/08/06 07:51:48 New Prometheus metric registered: queries_cached
pi-hole-exporter    | 2019-08-06T07:51:48.243009119Z 2019/08/06 07:51:48 New Prometheus metric registered: clients_ever_seen
pi-hole-exporter    | 2019-08-06T07:51:48.243182319Z 2019/08/06 07:51:48 New Prometheus metric registered: unique_clients
pi-hole-exporter    | 2019-08-06T07:51:48.243341386Z 2019/08/06 07:51:48 New Prometheus metric registered: dns_queries_all_types
pi-hole-exporter    | 2019-08-06T07:51:48.243514470Z 2019/08/06 07:51:48 New Prometheus metric registered: reply
pi-hole-exporter    | 2019-08-06T07:51:48.243689330Z 2019/08/06 07:51:48 New Prometheus metric registered: top_queries
pi-hole-exporter    | 2019-08-06T07:51:48.243860447Z 2019/08/06 07:51:48 New Prometheus metric registered: top_ads
pi-hole-exporter    | 2019-08-06T07:51:48.244019047Z 2019/08/06 07:51:48 New Prometheus metric registered: top_sources
pi-hole-exporter    | 2019-08-06T07:51:48.244193067Z 2019/08/06 07:51:48 New Prometheus metric registered: forward_destinations
pi-hole-exporter    | 2019-08-06T07:51:48.244364824Z 2019/08/06 07:51:48 New Prometheus metric registered: querytypes
pi-hole-exporter    | 2019-08-06T07:51:48.244522514Z 2019/08/06 07:51:48 New Prometheus metric registered: status
pi-hole-exporter    | 2019-08-06T07:51:48.245078558Z 2019/08/06 07:51:48 Starting HTTP server
pi-hole-exporter    | 2019-08-06T07:51:58.251806682Z 2019/08/06 07:51:58 Unable to login to PI-Hole, got a HTTP status code response '307' instead of '302'
```

**Comment by eko** [comment]
Hi, it looks like you have a redirect (`307 Temporary Redirect`) and I think the exporter does not follow redirections. I will have a look on what's possible over the end of the week.

Thank you

**Comment by eko** [comment]
Hi,

A quite similar issue has been fixed in new 0.0.5 release with new PI-Hole v4.3.2, you can grab the new binary here: https://github.com/eko/pihole-exporter/releases/tag/0.0.5

Closing this issue as if it still an issue, this is not related to the exporter itself.

Thank you
